UPGRADING.md

```diff
@@ -259,11 +280,11 @@ the full RPC interface provided as direct function calls. Import the
 the node service as in the following:
 
 ```go
-    node := node.NewDefault() //construct the node object
-    // start and set up the node service
+	node := node.NewDefault() //construct the node object
+	// start and set up the node service
 
-    client := local.New(node.(local.NodeService))
-    // use client object to interact with the node
+	client := local.New(node.(local.NodeService))
+	// use client object to interact with the node
```

Comment:
As above (split because GitHub):

```suggestion
// Construct a local (in-memory) RPC client to the node.
client := local.New(logger, node.(local.NodeService))
```

It's pre-existing but regrettable that we forced clients to type-assert here.